core/trino-main/src/main/java/io/trino/sql/planner/planprinter/PlanPrinter.java

```diff
@@ -1639,6 +1641,7 @@ else if (node.getScope() == Scope.LOCAL) {
                 addNode(node,
                         format("%sExchange", UPPER_UNDERSCORE.to(CaseFormat.UPPER_CAMEL, node.getScope().toString())),
                         ImmutableMap.of(
+                                "partitionCount", node.getPartitioningScheme().getPartitionCount().map(String::valueOf).orElse(""),
```

Comment:
`orElse("")` - for sure?

Reply:
It seems like a hack but TextRenderer skips empty string while printing the plan